package.json

```diff
@@ -36,8 +36,11 @@
     }
   },
   "lint-staged": {
-    "{packages,.storybook,scripts}/**/*.{js,ts,tsx,vue,md} *.js": [
+    "{packages,.storybook,scripts}/**/*.{js,ts,tsx,vue,md}": [
```

Comment:
Wait, what? I didn't recall how this gets committed, but thanks for sporting out!